tests/tracer/test_ext.py

```diff
@@ -21,7 +25,155 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    """Create temporary git directory, with one added file commit with a unique author and committer."""
+    cwd = str(tmpdir)
+    subprocess.check_output("git init", cwd=cwd, shell=True)
+    subprocess.check_output('git remote add origin "git@github.com:test-repo-url.git"', cwd=cwd, shell=True)
+    # Set committer user to be "Jane Doe"
+    subprocess.check_output('git config --local user.name "Jane Doe"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.email "jane@doe.com"', cwd=cwd, shell=True)
+    subprocess.check_output("touch tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output("git add tmp.py", cwd=cwd, shell=True)
+    # Override author to be "John Doe"
+    subprocess.check_output(
+        'GIT_COMMITTER_DATE="2021-01-20T04:37:21-0400" git commit --date="2021-01-19T09:24:53-0400" '
+        '-m "this is a commit msg" --author="John Doe <john@doe.com>" --no-edit',
+        cwd=cwd,
+        shell=True,
+    )
+    yield cwd
+
+
+@pytest.fixture
+def git_repo_error(tmpdir):
```

Comment:
```suggestion
def git_repo_empty(tmpdir):
```

What do you think? Since this isn't necessarily an "error" only repo, it is just empty, and therefore the commands we run will fail.

This is being pedantic, so not a blocker.